digest/src/buffer_macros/fixed.rs

```diff
@@ -0,0 +1,330 @@
+/// Creates a buffered wrapper around block-level "core" type which implements fixed output size traits.
+#[macro_export]
+macro_rules! buffer_fixed {
+    (
+        $(#[$attr:meta])*
+        $v:vis struct $name:ident$(<$gp:ident: $bound:ident>)?($core_ty:ty);
```

Comment:
We could simplify the macro by removing the `$(<$gp:ident: $bound:ident>)?` part. Currently it's used only by `gost94` and `skein`.

My initial intent was for this macro to support cases covered by the `buffer_ct_variable` macro, but, unfortunately, the required pattern becomes annoyingly complex, so I decided against it.